packages/storybook/stories/Examples/TeamsInterop/snippets/Lobby.snippet.tsx

```diff
@@ -78,3 +90,9 @@ export const Lobby = (props: LobbyProps): JSX.Element => {
     </VideoTile>
   );
 };
+
+export const decodeHtmlEntity = (str: string): string => {
```

Comment:
Had to add this little decode function cause `'` was being displayed encoded

Reply:
What is the `'` being encoded to and what strings does this regex replace? Can the problem be solved with encodeURI() or decodeURI() or is it a different issue?

Reply:
@allenplusplus Showing what it looks like without my decode function

<img width="2160" alt="Screen Shot 2021-05-31 at 2 43 10 PM" src="https://user-images.githubusercontent.com/9044372/120244344-d1d2e980-c21e-11eb-88d0-1f10b9e22570.png">

Reply:
Hey Josh, I looked into this a bit and this is actually a text knob issue as discussed here https://github.com/storybookjs/storybook/issues/4445. As aminimalanimal suggested, this change in Lobby.stories.tsx should work
```suggestion
export default {
  id: `${EXAMPLES_FOLDER_PREFIX}-teamsinterop-lobby`,
  title: `${EXAMPLES_FOLDER_PREFIX}/Teams Interop/Lobby`,
  component: Lobby,
  parameters: {
    docs: {
      page: () => getDocs()
    },
    knobs: {
      escapeHTML: false
    }
  }
} as Meta;
```

Reply:
Thanks @JoshuaLai , some digging found this: https://github.com/storybookjs/storybook/issues/4445. Looks like there is a way to disable the encoding using a storybook option. Perhaps we can use this but if it doesn't work I would say your solution is fine.

On the topic of should we allow unescaped string, I think it should be fine because React will treat all string plain text by default.

Reply:
Haha me and Allen found the same discussion

Reply:
👍 Awesome Thanks for the links let me try this!